test/util/reorg.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const assert = require('./assert');
```

Comment:
We should move towards using `bsert` over `test/util/assert`, that was a recent change to `hsd` where `test/util/assert` was removed

Reply:
There are some additional assertions that are specific to test cases, for example `asyncThrows` as well as others.

Reply:
There is a native function `assert.rejects` that was added to replace `asyncThrows`

https://nodejs.org/api/assert.html#assert_assert_rejects_asyncfn_error_message

The only problem is that it was added in Nodejs 10, but I don't see a good reason as to why we should continue to support lower versions as 9 has the memory issues https://github.com/bcoin-org/bcoin/issues/754 and 10 has a bunch of performance gains over 8/9

- https://v8.dev/blog/v8-release-66
- https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#2018-04-24-version-1000-current-jasnell
- https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V9.md#2017-12-12-version-930-current-mylesborins

Reply:
See https://github.com/handshake-org/hsd/commit/46334467b9b3a3ea379c7a7395cb5afa2aad0efa

Reply:
That is out-of-scope of this pull request.

Reply:
Furthermore, there may be additional test scenarios that require specific assertions.